FinancialTransactionsApi.Tests/V1/E2ETests/Fixture/TransactionDetailsFixture.cs

```diff
@@ -171,6 +143,7 @@ public void GivenAUpdateTransactionRequest()
         {
             var transaction = new TransactionDbEntity
             {
+                Pk = "#lbhtransaction",
```

Comment:
What is pk? I've noticed this is used as the HASH key name - could you please confirm the use of this property?

Reply:
Pk stands for PartitionKey, require to query that table